lib/util/config-utils.js

```diff
@@ -4,7 +4,9 @@ var Configstore = require('configstore');
 
 function initConfigFile(foreverRoot) {
   try {
-    return new Configstore(JSON.parse(fs.readFileSync(path.join(foreverRoot, 'config.json'))));
+    return new Configstore('config', undefined, {
```

Comment:
What's the idea behind this change?

Reply:
before this nconf was used, like this:
```
function initConfigFile(foreverRoot) {
  return new nconf.File({file: path.join(foreverRoot, 'config.json')});
}
```
whatever changes you made in the configuration, it was saved eventually in this config.json file.
The first param should be an `id` and then the `defaults` and then `options`. It passed the parsed json to the first param before, which was used for creating the config file appending a ".json" to it so it became '[object Object].json' in `~/.config/configstore` dir.
No backward compatibility, default configs was saved to this oddly named json and config.json in .forever dir was completely ignored.
I made it backward compatible and to use the original config file location.